opal/mca/accelerator/accelerator.h

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (c) 2014-2021 Intel, Inc. All rights reserved.
+ * Copyright (c) 2015      Los Alamos National Security, LLC. All rights
+ *                         reserved.
+ * Copyright (c) 2017-2022 Amazon.com, Inc. or its affiliates.
+ *                         All Rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#ifndef OPAL_ACCELERATOR_H
+#define OPAL_ACCELERATOR_H
+
+#include "opal/types.h"
+
+#include "opal/mca/mca.h"
+
+BEGIN_C_DECLS
+
+/**
+ * Accelerator flags
+ */
+#define MCA_ACCELERATOR_UNIFIED_MEMORY 0x00000001
```

Comment:
please add documentation of what this flag means.  If these are flags to be used in the accelerator `flags` fields throughout this header, please add `FLAGS` somewhere in this name.

Reply:
Updated with:
```
/**
 * Accelerator flags
 */
/* Unified memory buffers */
#define MCA_ACCELERATOR_FLAGS_UNIFIED_MEMORY 0x00000001
```